packages/nodes-base/nodes/MongoDb/MongoDb.node.ts

```diff
@@ -208,6 +210,10 @@ export class MongoDb implements INodeType {
 						query = query.sort(sort);
 					}
 
+					if (projection) {
```

Comment:
Does this need a similar check to sort?

Reply:
I don't think this was required for sort variable in the first place. But I want to focus on my feature and I don't feel "legit" to improve the existing code